package.json

```diff
@@ -30,6 +32,14 @@
 		"eject": "react-scripts eject",
 		"format": "prettier --write \"{src,__{tests,mocks}__}/**/*.jsx\""
 	},
+	"jest": {
+		"collectCoverageFrom": [
+			"src/**/*.{js,jsx,ts,tsx}",
+			"!<rootDir>/node_modules/",
+			"!<rootDir>/src/index.js",
```

Comment:
Hmm, I'm not so sure about ignoring this file actually. The appropriate way to test it would be to do an end-to-end test, since we can't directly test it with the `@testing-library/react` library. Although, seems kinda pointless to integrate an end-to-end testing tool just for a single file...